File: Recruit.Domain/Email.cs
using System.ComponentModel.DataAnnotations;

namespace Recruit.Domain
{
    public class Email
    {
        [Required]
        public string Name { get; set; }

        [Required, DataType("Email")]
        public string ReplyEmail { get; set; }
                
        public string Tel { get; set; }

        [Required]
        public string Enquiry { get; set; }
    }
}

File: Recruit.Web/Js/domain/Email.ts
module Admin.Domain {
    export interface IEmail{
        Name: string;
        ReplyEmail: string;
        Tel: string;
        Enquiry: string;
    }

    export class Email implements IEmail{
        public Name: string;
        public ReplyEmail: string;
        public Tel: string;
        public Enquiry: string;

        constructor() { };
    }
}